rclcpp_examples/CMakeLists.txt

```diff
@@ -12,12 +12,14 @@ find_package(rclcpp REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rmw_implementation REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
 
 ament_package()
 
 set(target_dependencies
   "rclcpp"
   "std_msgs"
+  "sensor_msgs"
```

Comment:
Since this dependency is only used for one executable please add it directly there.